clang/lib/Driver/ToolChain.cpp

```diff
@@ -230,6 +230,17 @@ static void getAArch64MultilibFlags(const Driver &D,
     Result.push_back(BranchProtectionArg->getAsString(Args));
   }
 
+  const Arg *StrictAlignArg = Args.getLastArgNoClaim(
+      options::OPT_mstrict_align, options::OPT_mno_unaligned_access);
+  if (StrictAlignArg) {
+    Result.push_back(StrictAlignArg->getAsString(Args));
+  }
+
+  const Arg *BigEndian = Args.getLastArgNoClaim(options::OPT_mbig_endian);
```

Comment:
I believe the same logic applies for `-mbig-endian` vs `-mlittle-endian`, where the last option provided takes effect over others.

Reply:
Done